mod-vendors/schemas/vendor.json

```diff
@@ -306,7 +330,7 @@
               "type": "boolean"
             },
             "send_to_emails": {
-              "description": "The email addresses to notify when this EDI job runs",
+              "description": "The comma delimited list of email addresses to notify when this EDI job runs",
```

Comment:
I'm not going to hold this up for this, but we should make sure we an example of this in the schema example files?